debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/SourceInfo.java

```diff
@@ -99,6 +101,7 @@
                                                      .field(TIMESTAMP_KEY, Schema.OPTIONAL_INT64_SCHEMA)
                                                      .field(TXID_KEY, Schema.OPTIONAL_INT64_SCHEMA)
                                                      .field(LSN_KEY, Schema.OPTIONAL_INT64_SCHEMA)
+                                                     .field(XMIN_KEY, Schema.OPTIONAL_INT64_SCHEMA)
```

Comment:
This probably needs to go to the end of the block, so to avoid backwards-compatibility issues when using Avro and the schema registry. @jpechane, can you confirm as per your exploration on the other PR?